src/user/UsersTabs.tsx

```diff
@@ -46,6 +46,7 @@ const UsersTabs = () => {
   const [user, setUser] = useState<UserRepresentation>();
   const [bruteForced, setBruteForced] = useState<BruteForced>();
   const [addedGroups, setAddedGroups] = useState<GroupRepresentation[]>([]);
+  const [forceFetch, setForceFetch] = useState(false);
```

Comment:
We're already using a pattern for this that uses a count to force a refresh. Let's keep that aligned with what we are already doing.

```suggestion
  const [refreshCount, setRefreshCount] = useState(0);
  const refresh = () => setRefreshCount((count) => count + 1);
```

Reply:
It's ok for me, I test with the change and it's still working